src/charts/tooltip.js

```diff
@@ -400,15 +405,68 @@ define(function(require){
                 .attr('x2', tooltipWidth - 60);
         }
 
+        /**
+         * Calculates the number of lines the tooltip title will need and updates the
+         * initialTooltipBodyYPosition accordingly
+         */
+        function updateTooltipTitleYPosition() {
+            const approximateNumberOfTitleLines = getApproximateNumberOfLines();
+            if(approximateNumberOfTitleLines > 1) {
+                additionalTooltipTitleHeight = 17 * (approximateNumberOfTitleLines -1)
+            }
+        }
+
+        /**
+         * Heuristic which gets the number of lines needed to display the title of the tooltip
+         * If shouldShowDateInTitle is set to true, it takes the formatted Date.now() as additional influencer
+         * for the approximation of the needed number of lines.
+         * @return  {Number}  approximateLineNumber  Approximative number of lines needed to display the title
+         * @private
+         */
+        function getApproximateNumberOfLines() {
```

Comment:
Maybe if you could make this function more generic we could add it to our text helper?

Reply:
Moved it and added a test for it.